src/core/directives/google-map.ts

```diff
@@ -79,6 +79,11 @@ export class SebmGoogleMap implements OnChanges, OnInit {
   zoom: number = 8;
 
   /**
+   * Enables/disables if map is draggable.
+   */
+  draggable: boolean = true;
```

Comment:
This attribute name looks good, but the name of the input should be mapDraggable (as we don't want to remap native DOM properties)